tycho-compiler-plugin/src/test/java/org/eclipse/tycho/osgicompiler/test/OsgiCompilerTest.java

```diff
@@ -210,6 +210,15 @@ private void assertSameFile(File expected, File actual) throws IOException {
         assertEquals(expected.getCanonicalFile(), actual.getCanonicalFile());
     }
 
+    public void testCopyResourcesNoConf() throws Exception {
+        File basedir = getBasedir("projects/resources/p004");
+        List<MavenProject> projects = getSortedProjects(basedir, null);
+        MavenProject project = projects.get(0);
+        getMojo(projects, project).execute();
+        assertTrue(new File(project.getBasedir(), "target/classes/testresources/Test.class").canRead());
+        assertTrue(new File(project.getBasedir(), "target/classes/testresources/test.properties").canRead());
```

Comment:
as this file is not mentioned in bin.includes should it really be copied?

Reply:
Yes, the expected behavior is that the resources that are part of source folders are copied into the same folder as .class files.

Reply:
You right I was confused by the path :-\